docs/extensions/netsuite/export-configuration.md

```diff
@@ -11,6 +11,8 @@ import TabItem from '@theme/TabItem';
 
 This page showcases the most common configurations. The final configuration depends heavily on the NetSuite instance configuration and might need to be adjusted as needed.
 
+When building the configuration, consult the [methods documentation](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_N3478008.html#Web-Services-Standard-Operations) and [schema browser](https://system.netsuite.com/help/helpcenter/en_US/srbrowser/Browser2022_2/script/record/vendor.html).
+
```

Comment:
Consider following with this stylistic change we use on the imports page: https://github.com/rossumai/university/blob/8e90c20e8e3e8d9d8be9077eadb9f1e746f23d63/docs/extensions/netsuite/import-configuration.md?plain=1#L21-L25